config/field.field.node.job.field_career_categories.yml

```diff
@@ -12,7 +12,7 @@ entity_type: node
 bundle: job
 label: 'Career categories'
 description: ''
-required: false
+required: true
```

Comment:
The ocha_content_classification module takes care of disabling the requirement when saving so reverting to the original value here.